scripts/bot-avatars.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+from upload import upload_task
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--force', dest='force', action="store_true", default=False)
+args = parser.parse_args()
+
+description = """Zulip allows users to create custom bots and integrations, and those bots need avatars to help make the product more fun! Zulip would like to provide a library of cute custom avatars that users can choose from when creating a new bot. Our non-custom avatars look like https://chat.zulip.org/integrations/, basically the logos of products that Zulip integrates with.
+
+The avatars should look good at the 50x50px size at which the user avatars are displayed in the Zulip web application. Use your creativity! For example, an avatar for a bot for logging software could be designed based on either its function or a pun around the word “logging”, e.g.:
```

Comment:
Let's put this and most of the rest of the description in the .md file text, so that we can change it as we get more info about what's confusing, etc. user-documentation.{md,py} might be a good one to look at (in a few minutes, I'll push a new version)